src/geometry/poly3/isA.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Determin if the given object is a 3D polygon.
+ * @params {poly3} object - the object to interogate
+ * @returns {true} if the object matches a poly3 based object
+ */
+const isA = (object) => {
```

Comment:
same remark as for others, although having a semi 'generic' operand name like this can be nice, not sure

Reply:
i like this idea better then having a utilitiy function that does a determination. the object itself knows best.

Reply:
not sure I understand ?

Reply:
There were functions called isCSG and isCAG in the utilities. I think these functions should be built into the objects, i.e. isA()

Reply:
ah yes I see your point !
hmm so we would have geom2.isA(xxx) that feels strange right ?
geom2.isOfThisType ? (meh)
isGeom2() feels more natural but you loose the reusability
then again clarity is better ?
not sure what naming is best to be honest

Reply:
I kind of comes down to this...

Do we want uniquely named utility functions? isGeom2, isPath2, etc

OR

Do we want to use the namespace to call isA()? geom2.isA(), path2.isA(), etc.

You might be happier with names like geom2.isCompatible() or geom2.isEquivalentType() or geom2.isUsableAs()